servers/publikator/lib/Notifications.js

```diff
@@ -29,18 +29,20 @@ const notifyPublish = async (repoId, context) => {
   const subscriptionDoc = await loaders.Document.byRepoId.load(formatRepoId)
   const subscriptionRepoId = subscriptionDoc.meta.repoId
 
-  const subscriptions = await getSubscriptionsForUserAndObject(
-    null,
-    {
-      type: 'Document',
-      id: subscriptionRepoId
-    },
-    context,
-    {
-      onlyEligibles: true
-    }
-  )
-  const subscribers = await Promise.map(
+  const subscriptions = testUsers
+    ? null
+    : await getSubscriptionsForUserAndObject(
+      null,
+      {
+        type: 'Document',
+        id: subscriptionRepoId
+      },
+      context,
+      {
+        onlyEligibles: true
+      }
+    )
+  const subscribers = testUsers || await Promise.map(
```

Comment:
(3) `testUsers` may come in truthy but not an array and I think it would be sane to check if `testUsers` is an array or not.